### PR TITLE
Support null_or_empty function

### PR DESCRIPTION
**Pull request by Lishi-hub**
It returns true if the string is an empty string or NULL. Otherwise it returns false.
#2592 #2593

### CODE REVIEW

File: docs/documentation/cn/sql-reference/sql-functions/string-functions/null_or_empty.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+|                   1 |
++---------------------+
+
+MySQL [(none)]> select null_or_empty("");
++-------------------+
+| null_or_empty('') |
++-------------------+
+|                 1 |
++-------------------+
+1 row in set (1.84 sec)
```
Comment (HangyuanLiu) [inline docs/documentation/cn/sql-reference/sql-functions/string-functions/null_or_empty.md]:
```suggestion
```

File: docs/documentation/en/sql-reference/sql-functions/string-functions/null_or_empty.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+|                   1 |
++---------------------+
+
+MySQL [(none)]> select null_or_empty("");
++-------------------+
+| null_or_empty('') |
++-------------------+
+|                 1 |
++-------------------+
+1 row in set (1.84 sec)
```
Comment (HangyuanLiu) [inline docs/documentation/en/sql-reference/sql-functions/string-functions/null_or_empty.md]:
```suggestion
```

**Review by HangyuanLiu — CHANGES_REQUESTED** [review]

File: docs/documentation/cn/sql-reference/sql-functions/string-functions/null_or_empty.md (hunk 1 above)
Comment (Lishi-hub) [reply docs/documentation/cn/sql-reference/sql-functions/string-functions/null_or_empty.md]: Modified

File: docs/documentation/en/sql-reference/sql-functions/string-functions/null_or_empty.md (hunk 2 above)
Comment (Lishi-hub) [reply docs/documentation/en/sql-reference/sql-functions/string-functions/null_or_empty.md]: Modified

File: docs/documentation/cn/sql-reference/sql-functions/string-functions/null_or_empty.md (hunk 1 above)
Comment (Lishi-hub) [reply docs/documentation/cn/sql-reference/sql-functions/string-functions/null_or_empty.md]: Modified

File: docs/documentation/en/sql-reference/sql-functions/string-functions/null_or_empty.md (hunk 2 above)
Comment (Lishi-hub) [reply docs/documentation/en/sql-reference/sql-functions/string-functions/null_or_empty.md]: Modified

**Comment by HangyuanLiu** [comment]
LGTM

**Review by kangkaisen — APPROVED** [review]
+1. LGTM

**Review by HangyuanLiu — APPROVED** [review]

**Comment by morningman** [comment]
It returns true if the string is empty or NULL. Otherwise it returns false.

**Pull request merged**